src/hooks/usePlaceOrder.ts

```diff
@@ -60,7 +61,6 @@ export const usePlaceOrder = (): Result => {
         return { success: false }
       }
 
-      setIsSubmitting(true)
```

Comment:
Should the behaviour also change for liquidity?

Even though the answer might be `no`, I think the other function (`placeMultipleOrders`) should match the change on this, because we'll soon be using that to place orders (when setting custom `from` date)

Thus I'd say to also update that function and liquidity code.

Reply:
ok, makes sense